config/300-serving.yaml

```diff
@@ -739,6 +739,47 @@ spec:
                               type: array
                           type: object
                       type: object
+                    resources:
+                      description: If specified, the container's resources.
+                      items:
+                        description: The pod this Resource is used to specify the requests and limits for
+                          a certain container based on the name.
+                        properties:
+                          container:
+                            description: The name of the container
+                            type: string
+                          limits:
+                            properties:
+                              cpu:
+                                pattern: ^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$
+                                type: string
+                              ephemeral-storage:
```

Comment:
serving system pods do not use `ephemeral-storage` so this should be dropped.

Reply:
it is defined in spec.resources already as well.

Reply:
but they were never evaluated or used, so starting to ignore them would not harm as they have been already ignored all the time (if already given). So removing it is not really removing a field, it's a clean-up. If someone has used that, then for some reason and she would be surprised that it has no effect at all.

I'm +1 for only adding the fields that are actually used, and adding more in the future if needed.